paasta_tools/api/api_docs/swagger.json

```diff
@@ -1124,6 +1140,24 @@
                 "bounce_method"
             ]
         },
+        "KubernetesPod": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "name of the pod in Kubernetes"
+                },
+                "deployed_timestamp": {
+                    "type": "number",
+                    "format": "float",
+                    "description": "Time at which the pod was deployed"
+                },
+                "phase": {
+                    "type": "string",
+                    "description": "The status of the pod"
+                }
+            }
+        },
```

Comment:
Add a container section with logs?

Reply:
Yeah, a container section should belong to `-vv` or `-vvv` option imo. I think we will definitely add it later. Just don't want to do everything in this PR. Otherwise, this gonna be very huge.